### ISSUE TITLE
Speactral javascript - New Document from yaml throw error

### ISSUE THREAD

**Issue by zlil** [stoplightio/spectral#2069]
Hey,

I'm trying to create a new Document from YAML string and I am receiving and error:
Cannot read properties of undefined (reading 'Yaml')

I've double-checked the string i'm receiving in online YAML linters and no errors were found.

```
const JSYAML = require('js-yaml')
const parseYamlToDoc = (yamlString) => {
 try { 
 JSYAML.load(yamlString) // this line is passed successfully, just need to check YAML is fine and no error thrown
return new Document(yamlString, Parsers.Yaml)
} catch (error) {
console.log(error) // here i'm receiving Cannot read properties of undefined (reading 'Yaml')
}
```

if I'm taking the exact same string and trying to parse it using 
`const yamlObj = parseYaml(specStr)`
everything works just fine.

so what's the problem actually with New Document, please?

**Comment by P0lip** [comment]
You presumably miss the import.
I don't see `const Parsers = require("@stoplight/spectral-parsers");`in your own code.

You can find a full example here
https://github.com/stoplightio/spectral/blob/develop/docs/guides/3-javascript.md#linting-a-yaml-string

**Comment by zlil** [comment]
Sorry, I omitted this line when I wrote the issue but I do import Parsers from '@stoplight/spectral-parsers' in my code so it must be something else.

**Comment by P0lip** [comment]
Could you share the entire code with me?

**Comment by zlil** [comment]
```
import Parsers from '@stoplight/spectral-parsers';
import { Spectral, Document } from '@stoplight/spectral-core';
import { oas } from '@stoplight/spectral-rulesets';
import JSYaml from "js-yaml";
import { parseYaml } from "@stoplight/spectral-parsers/dist/yaml";

export default class OpenApiService {

  private parseSpecToObject(specStr) {  
    try {
      JSYaml.load(specStr) 
      return new Document(specStr, Parsers.Yaml) // always throwing error
    } catch (error) {
      console.log(error)
    }
   
 try {
      JSYaml.load(specStr)
      return parseYaml(specStr); // works fine
    } catch (error) {
      console.log(error)
    }
   
 return null
  }

}
```

**Comment by P0lip** [comment]
try the following
`import * as Parsers from '@stoplight/spectral-parsers';`

**Comment by zlil** [comment]
> 

awesome, working. thanks.
